src/Resources/config/services.xml

```diff
@@ -111,5 +111,7 @@
             <argument id="service_container" type="service" />
         </service>
 
+        <service id="fin_search.sales_channel_context" synthetic="true" public="true"/>
```

Comment:
Can be removed after my previous comment is resolved

Reply:
See [my comment](https://github.com/findologic/plugin-shopware-6/pull/90#discussion_r466236349).